_extensions/quarto-svgbob/_extension.yml

```diff
@@ -1,7 +1,8 @@
 title: quarto-svgbob
-author: Dheepak Krishnamurthy
-  # - Fernando Martínez Santa
-version: 1.0.1
+author:
```

Comment:
```suggestion
authors:
```

Reply:
If you make it `authors` it should work fine

Reply:
Thanks a lot. I already did this change.